packages/jellyfish-network/src/TransactionSkipped.ts

```diff
@@ -0,0 +1,373 @@
+import { Network } from './Network'
+
+const MainNetSkippedTxId = new Set([
```

Comment:
```suggestion
const MainNetSkippedTxIds = new Set([
```
Wonder if this should assigned in plural form as its a Set of ids